_data/reference.yaml

```diff
@@ -116,67 +116,67 @@ toc:
       sidenav_disable_prefix_selection: true
       section:
       - title: pulumi
-        path: /reference/pkg/nodejs/@pulumi/pulumi/index.html
+        path: /reference/pkg/nodejs/@pulumi/pulumi/
```

Comment:
I pulled down your branch locally to test whether TOC selection still works with these changes, and looks like there's a subtle bug: If I select "cloudflare", both "cloud" and "cloudflare" show as selected.

<img width="257" alt="Screen Shot 2019-04-29 at 8 22 18 AM" src="https://user-images.githubusercontent.com/710598/56906837-e947b300-6a57-11e9-9674-5e5c6275c6a9.png">

The TOC selection code is a nightmare. Would love to burn it down and replace with something cleaner / more maintainable.

https://github.com/pulumi/docs/blob/master/_includes/toc.html
https://github.com/pulumi/docs/blob/master/_includes/toc_sub.html

Reply:
WE WILL KILL IT WITH FIRE